internal/journal/omap.go

```diff
@@ -74,7 +75,7 @@ func getOMapValues(
 
 	if err != nil {
 		if errors.Is(err, rados.ErrNotFound) {
-			util.ErrorLog(ctx, "omap not found (pool=%q, namespace=%q, name=%q): %v",
+			log.DebugLog(ctx, "omap not found (pool=%q, namespace=%q, name=%q): %v",
```

Comment:
```suggestion
			log.ErrorLog(ctx, "omap not found (pool=%q, namespace=%q, name=%q): %v",
```

Reply:
Thanks @pkalever, Covered in https://github.com/ceph/ceph-csi/pull/2433#discussion_r695543818 :+1: